frontend/src/utils/address.ts

```diff
@@ -147,6 +147,12 @@ const jsonFetch = (url: string) => fetch(url).then((res) => res.json());
 
 // Returns true if the address owns any POAP tokens
 const hasPOAPs = async (address: string) => {
-  const poaps = await jsonFetch(`https://api.poap.xyz/actions/scan/${address}`);
-  return poaps.length > 0 ? true : false;
+  try {
+    const poaps = await jsonFetch(`https://api.poap.xyz/actions/scan/${address}`);
+    return poaps.length > 0 ? true : false;
+  } catch (err) {
+    console.warn('Error in hasPOAPs');
+    console.warn(err);
```

Comment:
Reminder to always use `window.logger` in the frontend

Reply:
Noted! Addressed it here https://github.com/ScopeLift/umbra-protocol/pull/370/commits/ff0b19c2d6e044393dee68cdf5716548d8d5f884